pkgs/top-level/release-lib.nix

```diff
@@ -43,6 +46,11 @@ rec {
   pkgs_x86_64_cygwin = allPackages { system = "x86_64-cygwin"; };
 
 
+  assertTrue = bool:
+    if bool
+    then pkgs.runCommand "evaluated-to-true" {} "touch $out"
+    else pkgs.runCommand "evaluated-to-false" {} "false";
```

Comment:
Why is this? Why not just use `assert`?

Reply:
@Ericson2314 so what is easier, an extra job to test these, or changing Hydra so it tracks evaluation errors by revision?

Reply:
Hmm? In a later PR I rewrote the tests the semi-recursive-Nix way.